### DISCUSSION TITLE
Contributing to the Project

### DISCUSSION THREAD

**Discussion by theCJMan** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#72] [Q&A] [answered]
Hi there
I was wondering if you are looking for more contributors, I have been playing with the SAMPLE directory/ project and there are several items I would like to add / adapt..

My project would follow its own rout, but was wondering if you wanted any more Contributions..
If not, no issues

**Comment by gabrieldwight** [comment, marked as answer]
Hi, you can contribute to the project if you discover a bug that needs to be fixed and other features you have implemented that are not included in the library. Please feel free to send a PR and I will look at it.
